src/PrestaShopBundle/Install/Install.php

```diff
@@ -988,8 +988,12 @@ public function installModules(array $modules): bool
 
         ModuleEntity::updateTranslationsAfterInstall(false);
 
+        $tmp = array_filter($modules, function ($module) {
```

Comment:
`$tmp` name should be more explicit

Reply:
This file shouldn't be push, it's a modification I had to make because I had a problem installing modules when I was using docker-compose